NuKeeper/Configuration/UserSettings.cs

```diff
@@ -10,16 +10,8 @@ public class UserSettings
     {
         public NuGetSources NuGetSources { get; set; }
 
-        public Regex PackageIncludes { get; set; }
-
-        public Regex PackageExcludes { get; set; }
-
-        public int MaxPackageUpdates { get; set; }
-
         public int MaxRepositoriesChanged { get; set; }
 
-        public TimeSpan MinimumPackageAge { get; set; }
-
         public VersionChange AllowedChange { get; set; }
```

Comment:
Shouldn't `AllowedChange` also belong on the `PackageFilter`?

Reply:
`FilterSettings` is currently all the fields used in `UpdateSelection. `AllowedChange` is used elsewhere. It's purely locality of code rather than being conceptually related.